src/main/java/blusunrize/immersiveengineering/common/blocks/ItemBlockIEStairs.java

```diff
@@ -8,26 +8,33 @@
 
 package blusunrize.immersiveengineering.common.blocks;
 
-import blusunrize.immersiveengineering.api.Lib;
 import net.minecraft.block.Block;
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.Item;
+/*
+import blusunrize.immersiveengineering.api.Lib;
 import net.minecraft.client.resources.I18n;
 import net.minecraft.client.util.ITooltipFlag;
-import net.minecraft.creativetab.CreativeTabs;
-import net.minecraft.item.BlockItem;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.NonNullList;
+import net.minecraft.util.text.ITextComponent;
 import net.minecraft.world.World;
-
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
 import javax.annotation.Nullable;
 import java.util.List;
+*/
 
+//TODO This class should be obsolste by now. The tooltip can be handled in `IEStairsBlock`.
+// 		 Commented out for main author review/removal.
```

Comment:
Same as for `ItemBlockIESlabs`.

Reply:
Same here, I also think it's not needed anymore.